sources/Inputs/src/NetworkManager.cpp

```diff
@@ -57,9 +57,10 @@ NetworkManager::buildTree() {
   const auto& voltageLevels = network->getVoltageLevels();
   for (auto it_v = voltageLevels.begin(); it_v != voltageLevels.end(); ++it_v) {
     if ((*it_v)->getVoltageLevelTopologyKind() == DYN::VoltageLevelInterface::NODE_BREAKER) {
-      LOG(error) << MESS(NodeBreakerInterface, (*it_v)->getID()) << LOG_ENDL;
-      // This is done in class constructor so we exit directly
-      std::exit(EXIT_FAILURE);
+      // Do not emit error if a node/breaker topology is found
+      // A version of Dynawo that supports node/breaker is required
+      // See https://github.com/dynawo/dynawo/pull/1536
```

Comment:
We generally avoid referencing github issues in the code.
Did you test that the main connex component is correct in the case node_breaker?

Reply:
Reference removed.
What do you main by testing that main connex component is correct ?

Reply:
thanks!
I mean that there is an algorithm lused ater in this code that go through the network to detect which nodes belong to the main component connex. I just wanted to make sure that it still works when node breaker nodes are involved.